Contributors-Guide/GlusterFS-Release-process.md

```diff
@@ -32,42 +121,33 @@ The tracker bugs are used as guidance for blocker bugs and should get created wh
 - you should now be able to use the 'glusterfs-a.b.c' to access the bug, use the alias to replace the BZ# in URLs, or '''blocks''' fields
 - bugs that were not fixed in this release, but were added to the tracker should be moved to the new tracker
 
-
-Create Release Announcement
----------------------------
-
-Create the Release Announcement (this is often done while people are
-making the packages). The contents of the release announcement can be
-based on the release notes, or should at least have a pointer to them.
+#### Create Release Announcement
+Create the Release Announcement (this is often done while people are making the packages). The contents of the release announcement can be based on the release notes, or should at least have a pointer to them.
 
 Examples:
+- [blog](http://blog.gluster.org/2014/11/glusterfs-3-5-3beta2-is-now-available-for-testing/)
+- [release notes](https://github.com/gluster/glusterfs/blob/v3.5.3/doc/release-notes/3.5.3.md)
+
+#### Send Release Announcement
+Once the Fedora/EL RPMs are ready (and any others that are ready by then), send the release announcement:
 
--   [blog](http://blog.gluster.org/2014/11/glusterfs-3-5-3beta2-is-now-available-for-testing/)
--   [release
-    notes](https://github.com/gluster/glusterfs/blob/v3.5.3/doc/release-notes/3.5.3.md)
+- Gluster Mailing lists
+  - gluster-announce, gluster-devel, gluster-users
+- Gluster Blog
+- Gluster Twitter account
+- Gluster Facebook page
+- Gluster LinkedIn group
+- Gluster G+
```

Comment:
I cannot find the link to the LinkedIn page.
I found a couple of pages, but don't know which is official.

https://www.linkedin.com/groups/6501354/profile has an official feel, but is currently a closed community and has less members.

https://www.linkedin.com/groups/99784 has way more members, but to me doesn't feel like the official group.